mmbexplorer/src/MMBE_Gui.cpp

```diff
@@ -626,9 +628,10 @@ void CMMBETable::draw_cell( TableContext context, int _row, int _col, int _x, in
 
 void CMMBETable::resize( int _x, int _y ,int _w ,int _h )
 {
+    int dcount = max(mMMB->GetNumberOfDisks() + 1,512);
```

Comment:
In order to compile on MacOS and Linux, I changed all "max(" and "min(" instances for "std::max(" and "std::min(" as max and min functions are included via the <algorithm> include file.

Also, the compiler is quite picky with the type of the operand on line 631, so I needed to add (size_t) before 512 on that line to force the type.

Reply:
> In order to compile on MacOS and Linux, I changed all "max(" and "min(" instances for "std::max(" and "std::min(" as max and min functions are included via the include file.
> 

Sorry I am not quite clear on what is required here, simply changing min and max to std::min and std:max results in errors. Can you explain further? my C++ experience is limited and from a long time ago.

> Also, the compiler is quite picky with the type of the operand on line 631, so I needed to add (size_t) before 512 on that line to force the type.

I have added this.